README.rst

```diff
@@ -413,6 +413,15 @@ Flask with uWSGI`_ (this is common):
     app = connexion.App(__name__, specification_dir='swagger/')
     application = app.app # expose global WSGI application object
 
+You can use the ``aiohttp`` framework as server backend as well:
+
+.. code-block:: python
+
+    import connexion
+
+    app = connexion.AioHttpApp(__name__, specification_dir='swagger/')
+    app.run(port=8080)
+
```

Comment:
Here we could add something like
**NOTE:** Also check aiohttp handler [examples](https://docs.aiohttp.org/en/stable/web.html#handler)